docs/registries.conf.5.md

```diff
@@ -0,0 +1,26 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`. You can list multiple registries using
```

Comment:
@nalid the block thing is only used by a different parser (aka atomic-registries) so the functions ro read and handle are actually there ... because we want to deprecate it, i never added a function into this parsing library.